src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -1166,18 +1155,17 @@
             "name": "ALICIA L. PRIMICIAS-ENRIQUEZ"
           },
           "vice_mayor": {
-            "name": "ALVIN O. BRAVO"
+            "name": "MARICON OPERAÑA"
```

```suggestion
            "name": "MARICON VINDY A. OPERAÑA"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=055&m=36